src/astra/astra-api.module.ts

```diff
@@ -0,0 +1,9 @@
+import { HttpModule, Module } from '@nestjs/common';
+import { KeyspaceService } from './keyspace.service';
+
+@Module({
+  imports: [HttpModule],
+  providers: [KeyspaceService],
+  exports: [KeyspaceService],
+})
+export class AstraApiModule {}
```

Comment:
Maybe this could be called `Astra` to match the folder structure, then if it clashes in the imports we could use an alias

Reply:
`AstraModule` already gets imported from my package, that's why I named it `AstraApi` but I think the alias should be no problem